teachbooks/external_content/headers.py

```diff
@@ -6,13 +6,28 @@
 
 def format_header(cfg: dict, base_url: str, version: str) -> str:
     """Generate the admonition header based on the user's config."""
-    admonition = (
-        "```{" + cfg["attribution_color"] + "} Attribution\n"
-        ":class: attribution\n"
-        f"This page originates from {base_url},"
-        f" version: {version}\n"
-        "```\n"
-    )
+    # Check if language is set to Dutch in the configuration
+    language = cfg.get("language", "en")
+
+    if language == "nl":
+        # Dutch attribution text
+        admonition = (
+            "```{" + cfg["attribution_color"] + "} Bronvermelding\n"
```

Comment:
```suggestion
            f"```{{{cfg['attribution_color']}}} Bronvermelding\n"
```
we can still use an f-string here.

If we would add more languages it would be nice to refactor this to something like
```py
            f"```{{{cfg['attribution_color']}}} {localization[lang]["attribution"]}\n"
```
would would remove the if/else block here. But that's probably a premature optimization.